src/ts/components/subtitlesettings/subtitlesettingselectbox.ts

```diff
@@ -41,4 +41,8 @@ export class SubtitleSettingSelectBox extends SelectBox {
       this.overlay.getDomElement().addClass(this.currentCssClass);
     }
   }
+
+  configure(player: PlayerAPI, uimanager: UIInstanceManager): void {
+      this.settingsManager = uimanager.subtitleSettingsManager;
```

Comment:
is there a chance of `settingsManager` getting accessed before `configure` is called?

Reply:
Not the way how it's currently implemented. However, when one changes the implementation, we'd need to be careful (that applies for any `localStorage` operations, as its safety depends on the timing).

I at least made the prop optional to indicate it could be undefined:
[Replce get accessor with regular method](https://github.com/bitmovin/bitmovin-player-ui/pull/615/commits/c5d5e4f8000621dd6bb1141d45aa557782b83c00)